cmake/ElektraCache.cmake

```diff
@@ -365,14 +365,8 @@ set (LIB_SUFFIX ""
 		"Optional suffix to use on lib folders (e.g. 64 for lib64)"
     )
 
-set (MEMORYCHECK_COMMAND
```

Comment:
> You are too fast :). I think I found a problem in 4c25e65…

I think I spoke to soon. I just forgot  to add `-DENABLE_TESTING=ON`. The following works on my machine – in a clean build directory – if I apply the changes contained in this pull request:

```sh
cmake .. -DENABLE_TESTING=ON
make
make run_memcheck
```

I guess something like `cmake .. -DENABLE_TESTING=ON cmake .. -DMEMORYCHECK_COMMAND…` might work too, but I prefer, that we just apply 4c25e65. 

Here are the reasons why:

1. It should work for most people, without the need to configure anything manually. The current solution only works for setups where `valgrind` is located in `/usr/bin/`.
2. Commit 4c25e65 removes code. Less code generally means less problems.

> strange, actually your -DMEMORYCHECK_COMMAND should take preference?

I did not specify `-DMEMORYCHECK_COMMAND` but just changed the hardcoded paths to `/usr/local/bin/valgrind` . I think I forgot to add `-DENABLE_TESTING=ON` when I tried this. So, it is quite possible that it would have worked.

> Or did you try the cmake GUI?

No.

> Nevertheless, simply put an if around it would be less invasive, but I think you can can override it even without.

That would mean even more code… :). If it works, even when we do not hardcode the path to `valgrind`, then I see no reason why we should to that…